src/components/media/__tests__/Media.test.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import { mount, shallow } from 'enzyme';
 
 import Media from '../Media';
 
-jest.mock('../../../icons/general/IconEllipsis', () => () => <icon-ellipsis />);
```

Comment:
Did this custom name throw a type error?

Reply:
Yup. 
> Property 'icon-ellipsis' does not exist on type 'JSX.IntrinsicElements'. ts(2339)

Reply:
Can you put a testID or something on there so the snapshot reflects that the icon is put in the right place?